servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Publisher.java

```diff
@@ -1344,6 +1321,166 @@ public final Completable flatMapCompletableDelayError(Function<? super T, ? exte
                 .ignoreElements();
     }
 
+    /**
+     * Map each element of this {@link Publisher} into a {@link Single}&lt;{@link R}&gt; and flatten all signals
+     * emitted from each mapped {@link Single}&lt;{@link R}&gt; into the returned {@link Publisher}&lt;{@link R}&gt;.
+     * Each mapped {@link Single}&lt;{@link R}&gt; maybe subscribed to concurrently but the results are emitted in
+     * the same order as this {@link Publisher}.
+     * <p>
+     * To control the amount of concurrent processing done by this operator see
+     * {@link #flatMapConcatSingle(Function, int)}.
+     * <p>
+     * This method is similar to {@link #map(Function)} but the result is asynchronous, and provides a data
```

Comment:
Perhaps contrast with `flatMapMergeSingle` or `@see flatMapMergeSingle`

Reply:
I'll add a `@see flatMapMergeSingle` for now to keep javadocs compressed.